tests/integration_tests/tags/commands_tests.py

```diff
@@ -112,6 +112,7 @@ def test_delete_tags_command(self):
                 TaggedObject.object_id == example_dashboard.id,
                 Tag.type == TagType.custom,
             )
+            .order_by(Tag.name)
```

Comment:
What does this change have to do with the redirect logic?

Reply:
Nothing. It was something I caught while running the tests. It's a flaky test given that order is not guaranteed.